HRTool/src/main/java/app/Application.java

```diff
@@ -33,7 +35,7 @@
 @SpringBootApplication
 
 public class Application extends SpringBootServletInitializer implements CommandLineRunner {
```

Comment:
Please remove the everything from the Application.class.

This should be the only thing in Application.class.

    public static void main(String[] args) {
        SpringApplication.run(Application.class);
    }

    @Override
    protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
        return application.sources(Application.class);
    }